src/data/faq_de.json

```
@@ -102,6 +102,159 @@
+                            "Das Impfzertifikat (QR-Code) wird z.B. über die CovPass App oder die Corona-Warn-App (CWA) digital oder alternativ als maschinenlesbarer Ausdruck genutzt. Das Impfzertifikat enthält nur Informationen zum Impfstatus, den Namen des Geimpften und das Geburtsdatum. Für Dienstleister, die den Impfstatus überprüfen möchten, wird es eine Prüf-App zur Prüfung des Impfzertifikats geben. Damit kann der Impfstatus ähnlich wie ein Barcode eines Flug- oder Bahntickets gescannt werden. Alternativ ist ein Nachweis mit dem analogen Impfpass möglich."
+                        ]
+                    },
+                    {
+                        "title": "Kann man auch im digitalen Impfnachweis speichern, dass man bereits infiziert war oder negativ getestet wurde?",
+                        "anchor": "vaccination_certificate_recovered_negative_test",
+                        "active": true,
+                        "textblock": [
+                            "Ja, auch negative Tests oder eine durchgemachte Infektion lassen sich in der CovPass- App und Corona-Warn-App als Testzertifikat bzw. Genesenenzertifikat hinterlegen. Eine durchgemachte Infektion wird mittels eines PCR-Test-Ergebnisses dokumentiert."
+                        ]
+                    },
+                    {
+                        "title": "Warum wurde kein EU-weites Projekt für ein Zertifikat ausgeschrieben?",
+                        "anchor": "vaccination_certificate_eu",
+                        "active": true,
+                        "textblock": [
+                            "Eine gemeinsame EU-Ausschreibung hätte zu viel Zeit benötigt und wäre aufgrund der unterschiedlichen Impfinformationssysteme in den Mitgliedstaaten auch schwierig umzusetzen gewesen. Beim EU-Ansatz geht es um die Regelung eines Anerkennungsrahmens. Etwaige grenzüberschreitende Nutzungspiloten könnten seitens der EU-Kommission begleitet werden. Bei der Umsetzung des Digitalen Impfnachweises in Deutschland werden die EU-Vorgaben von vornherein berücksichtigt."
+                        ]
+                    },
+                    {
+                        "title": "Wirkt sich der Impfstatus auf die Berechnung des Risikos aus? Beispielsweise, wenn eine geimpfte Person sich infiziert haben sollte und dann andere Nutzer*innen warnt?",
+                        "anchor": "vaccination_certificate_risk_assesment",
+                        "active": true,
+                        "textblock": [
+                            "Nein, in der Risikoberechnung wird nicht berücksichtig, ob Sie gegen Corona geimpft oder genesen sind. Die Funktionen sind voneinander unabhängig. Die Qualität Ihrer Risikobewertung ändert sich nicht, je nachdem, ob Sie Sich dafür entscheiden, Ihren Impfausweis in der Anwendung zu speichern, was unterstreicht, dass dies völlig freiwillig ist. Ihr persönlicher Impfstatus kann jedoch Ihre Interpretation der Risiko-Warnung beeinflussen."

```

Suggestion from https://github.com/corona-warn-app/cwa-website/issues/1311#issuecomment-850884993:

```suggestion
                            "Nein, in der Risikoberechnung wird nicht berücksichtig, ob Sie gegen Corona geimpft oder genesen sind. Die Funktionen sind unabhängig voneinander – wenn Sie sich entscheiden, Ihren Impfausweis nicht in der Anwendung zu speichern, wird dadurch die Genauigkeit Ihrer Risikobewertung nicht verschlechtert. Ihr persönlicher Impfstatus kann jedoch Ihre Interpretation der Risiko-Warnung beeinflussen."

```

---

okis

---

{
                        "titel": "Does my vaccination status affect the calculation of the risk? Does the calculation of the risk change if a vaccinated person has become infected, and then warns other users?",                        
                        "anchor": "vac_cert_risk_assesment",
                        "active": true,
                        "textblock": [
                            "Risk calculation and vaccination certificates are independent features of the app. The risk calculation does not take into account whether you are vaccinated against COVID-19. The calculation of the risk also does not change if a vaccinated person has become infected, and then warns other users."
                        ]
                    },

---

"title": "Wirkt sich mein Impfstatus auf die Berechnung des Risikos aus? Ändert sich die Berechnung des Risikos, wenn eine geimpfte Person sich infiziert haben sollte und dann andere Nutzer*innen warnt?",

                            "Risikoberechnung und Impfzertifikate sind unabhängig Funktionen der App. In der Risikoberechnung wird nicht berücksichtig, ob Sie gegen Corona geimpft sind. Die Berechnung des Risikos ändert sich auch nicht, wenn eine geimpfte Person sich infiziert haben sollte und dann andere Nutzer*innen warnt."